init-tools.sh

```diff
@@ -42,7 +43,7 @@ esac
 
 if [ ! -e $__PROJECT_JSON_FILE ]; then
     if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
-
+    echo "Running: $__scriptpath/init-tools.sh" > $__init_tools_log
```

Comment:
This script doesn't take any arguments so I don't believe this is needed. The one that takes arguments is the one in buildtools and we are logging those already.